src/connector/connection_info.rs

```diff
@@ -192,6 +192,20 @@ impl ConnectionInfo {
         }
     }
 
+    /// Whether the pgbouncer mode is enabled.
+    pub fn pg_bouncer(&self) -> bool {
+        match self {
+            #[cfg(feature = "postgresql")]
+            ConnectionInfo::Postgres(url) => url.pg_bouncer(),
+            #[cfg(feature = "mysql")]
+            ConnectionInfo::Mysql(_) => false,
+            #[cfg(feature = "mssql")]
+            ConnectionInfo::Mssql(_) => false,
+            #[cfg(feature = "sqlite")]
+            ConnectionInfo::Sqlite { .. } => false,
```

Comment:
shouldn't the last 3 cases just be `_ => false`? Would be easier to process visually imo.

Reply:
True